src/Symfony/Component/Uid/Command/GenerateUlidCommand.php

```diff
@@ -25,12 +25,8 @@
 #[AsCommand(name: 'ulid:generate', description: 'Generate a ULID')]
 class GenerateUlidCommand extends Command
 {
-    private UlidFactory $factory;
-
-    public function __construct(UlidFactory $factory = null)
+    public function __construct(private UlidFactory $factory = new UlidFactory())
```

Comment:
let's always put CPP properties on their own line, that'll reduce also the diff when adding new arguments in the future

Reply:
I like it 👍🏻

Reply:
Done